File: src/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using CefSharp;
using CefSharp.WinForms;


namespace SharpBrowser



{
    public partial class Form1 : Form




    {

        public ChromiumWebBrowser browser;
        public void InitBrowser()
        {
            Cef.Initialize(new CefSettings());
           // browser = new ChromiumWebBrowser("www.google.com");
            browser = new ChromiumWebBrowser("www.clarin.com");
            this.Controls.Add(browser);
           // browser.Dock = DockStyle.Fill;
           browser.Dock = DockStyle.Fill  ;
        }

        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            InitBrowser();
         //browser.Load("www.lanacion.com");
        }
        
    }
}

File: src/WindowsApplication1/Form1.vb
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms
Imports CefSharp
Imports CefSharp.WinForms


Namespace SharpBrowser



    Partial Public Class Form1
        Inherits Form





        Public browser As ChromiumWebBrowser
        Public Sub InitBrowser()
            Cef.Initialize(New CefSettings())
            browser = New ChromiumWebBrowser("www.google.com")
            Me.Controls.Add(browser)
            browser.Dock = DockStyle.Fill
        End Sub

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(sender As Object, e As EventArgs)
            InitBrowser()
        End Sub

        Private Sub InitializeComponent()
            Me.SuspendLayout()
            '
            'Form1
            '
            Me.ClientSize = New System.Drawing.Size(284, 261)
            Me.Name = "Form1"
            Me.ResumeLayout(False)

        End Sub

        Private Sub Form1_Load_1(sender As Object, e As EventArgs) Handles MyBase.Load

        End Sub
    End Class
End Namespace


